.github/workflows/integration.yaml

```diff
@@ -33,28 +34,44 @@ jobs:
         run: mvn -B clean verify
 
       - name: Login to Quay
-        if: github.event_name == 'push' && github.ref_name == 'main'
+        if: github.event_name == 'push'
```

Comment:
does this mean, it will login to quay every time PR is raised? I remember this fails when trigged via PR because of forked branch not having these secrets?

Reply:
From experimenting on my fork these should not be triggered by PRs

The workflows are still installed in my fork and PRs do not login to quay on PR creation or sync (pushing additional changes to the PR) https://github.com/robobario/flink-sql-examples/pull/11

the github `push` event is triggered by the refs being pushed. you can see on the pull request `Check` that the event causing it is `pull_request`.

Pushing the `test-pr2` branch to my fork doesn't trigger a workflow (because we restrict integration.yaml to operate on` main` and `release/**`). But the pull request is then synchronized and that triggers the pull_request synchronize event, which triggers integration.yaml.

Reply:
Cool, thanks for explaining.